src/cdk/scrolling/virtual-scroll-viewport.html

```diff
@@ -9,7 +9,4 @@
   Spacer used to force the scrolling container to the correct size for the *total* number of items
   so that the scrollbar captures the size of the entire data set.
 -->
-<div class="cdk-virtual-scroll-spacer"
-     [style.height.px]="orientation === 'horizontal' ? 1 : _totalContentSize"
-     [style.width.px]="orientation === 'horizontal' ? _totalContentSize : 1">
-</div>
+<div class="cdk-virtual-scroll-spacer" [style.transform]="_totalContentSizeTransform"></div>
```

Comment:
I might be a good idea to try this out on IE. I remember it doing some weird stuff when it comes to sizing and `transform`.